SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialLoadBalancingHandler.h

```diff
@@ -2,30 +2,49 @@
 
 #pragma once
 
-#include "EngineClasses/SpatialNetDriver.h"
 #include "EngineClasses/SpatialPackageMapClient.h"
-#include "Utils/SpatialActorUtils.h"
+#include "Schema/AuthorityIntent.h"
 
 DECLARE_LOG_CATEGORY_EXTERN(LogSpatialLoadBalancingHandler, Log, All);
 
+namespace SpatialGDK
+{
+class SpatialEventTracer;
+class FSubView;
+} // namespace SpatialGDK
+
+class USpatialNetDriver;
+
+enum class EActorMigrationResult : uint8
+{
+	Success,
+	NotAuthoritative,
+	NotReady,
+	PendingKill,
+	NotInitialized,
+	Streaming,
+	NetDormant,
+	NoSpatialClassFlags,
+	DormantOnConnection
+};
+
 class FSpatialLoadBalancingHandler
```

Comment:
Shouldn't this class be inside `SpatialGDK` namespace? 🤔

Reply:
yes

Reply:
MAYBE the enum shouldn't have been, but "migration" is a uniquely GDK concept so left it for now